uiowa/uiowa
# Admissions - AoS Search Improvements

Resolves #3258 

Visually approved by Admissions

## To Test

- pull
- `composer install && yarn workspace admissions_core build && blt ds --site=admissions.uiowa.edu`
- Test the search box on the front page. It should populated the exposed filter on the areas of study.
- The filters should all work if the content exists. You can change your admissions blt file to pull from dev if you want more content.

To those who saw the white screen in demo, manually trying to filter values via URL is not something you should do...
https://www.drupal.org/project/drupal/issues/2932604, https://www.drupal.org/project/drupal/issues/2901127
With that said, Drupal should really do something to prevent this from happening...
